apps/console/src/features/applications/components/settings/attribute-management/attribute-settings.tsx

```diff
@@ -1070,7 +1080,8 @@ export const AttributeSettings: FunctionComponent<AttributeSettingsPropsInterfac
             delete submitValue.claimConfiguration.subject;
         }
 
-        updateClaimConfiguration(appId, submitValue)
+        updateClaimConfiguration(appId, submitValue) &&
+        updateAuthProtocolConfig<OIDCDataInterface>(appId, oidcSubmitValue, SupportedAuthProtocolTypes.OIDC)
```

Comment:
Async functions should not be chained with &&.
If both can be executed concurrently, consider `Promise.all([ ])`

Reply:
fixed with [a5f0e39](https://github.com/wso2/identity-apps/pull/4284/commits/a5f0e399c3a9fa90a48a937face8fe7d2f554faf)